src/platforms/alexa/directives.ts

```diff
@@ -238,14 +238,14 @@ export class StopAudio implements IDirective {
   }
 }
 
-export class GameEngineStartInputHandler implements IDirective {
-  public static key: string = "alexaGameEngineStartInputHandler";
+export class GadgetControllerLightDirective implements IDirective {
+  public static key: string = "alexaGadgetControllerLightDirective";
   public static platform: string = "alexa";
 
-  public template: interfaces.gameEngine.StartInputHandlerDirective;
+  public lightDirective: any;
 
-  constructor(template: interfaces.gameEngine.StartInputHandlerDirective) {
-    this.template = template;
+  constructor(lightDirective: any) {
```

Comment:
let's use better types and perhaps a type clause 

https://www.typescriptlang.org/docs/handbook/advanced-types.html#user-defined-type-guards